### PR TITLE
tests: Implement VW PIN flow for test fixture

### PR DESCRIPTION
**Pull request by nbry**
Updated geckodriver flow to handle newly implemented Volkswagen Pin flow. Testing fixture should work now

Asana: https://app.asana.com/0/1200617420635813/1200661538477511

### CODE REVIEW

File: tests/auth_helpers.py [hunk 1]
```diff
@@ -86,6 +86,18 @@ def run_auth_flow(auth_url, brand="CHEVROLET"):
...
+    # For Volkswagen Pin Flow
+    if brand == "VOLKSWAGEN":
+        # Click the first vehicle compatible with EV endpoints. *Should* be an eGolf
+        driver.find_element_by_css_selector(".input-button-custom.checkbox").click()
+
+        # Enter Pin 1-1-1-1
+        driver.find_element_by_id("f2-pin-input-number-1").send_keys("1")
+        driver.find_element_by_id("f2-pin-input-number-2").send_keys("1")
+        driver.find_element_by_id("f2-pin-input-number-3").send_keys("1")
+        driver.find_element_by_id("f2-pin-input-number-4").send_keys("1")
```
Comment (s-ashwinkumar) [inline tests/auth_helpers.py:98]: nit: can you convert it into a loop and just have the `driver.find_element_by_id("f2-pin-input-number-<loop-value>").send_keys("1")` once ?

File: tests/auth_helpers.py [hunk 2]
```diff
@@ -86,6 +86,15 @@ def run_auth_flow(auth_url, brand="CHEVROLET"):
     driver.find_element_by_id("password").send_keys("password")
     sign_in_button.click()
 
+    # For Volkswagen Pin Flow
+    if brand == "VOLKSWAGEN":
+        # Click the first vehicle compatible with EV endpoints. *Should* be an eGolf
```
Comment (gurpreetatwal) [inline tests/auth_helpers.py:91]: This assumption is a bit "risky" in that it'll only be true if Connect was launched with `required:` for an EV permission. That being said I think it's fine as is, but maybe just note that it should be an eGolf if connect was launched with an EV permission as required.

Comment (nbry) [reply tests/auth_helpers.py:91]: Ah, yeah. I think I can do a few things then:

- Do an extra check on the element next to the check box and check if the string includes "E-Golf" (or something on those lines). At this point I can have the tests error and come to a halt, or just print a console warning run the tests without the egolf fixture.
- Alternatively, I can just write a detailed comment on [line 142 of the test configurations file](https://github.com/smartcar/python-sdk/blob/32128df569f2be5a9e19e941b7fd66a833a584ed/tests/conftest.py#L142), specifying that the fixture _must_ have an EV permission required. At the present, it's hard-coded to  "required:control_charge".

Comment (s-ashwinkumar) [reply tests/auth_helpers.py:91]: my 2 cents : I think the lowest hanging fruit if you don't want to drag this longer is just change use of a VW EV to a Tesla

**Comment by nbry** [comment]
Using a Tesla testing fixture instead of a VW E-golf now

**Review by gurpreetatwal — APPROVED** [review]

**Pull request merged**